src/WebJobs.Script/Utility.cs

```diff
@@ -305,13 +305,6 @@ public static JObject ToJObject(ExpandoObject value)
             return JObject.Parse(json);
         }
 
-        public static IJobHostMetadataProvider CreateMetadataProvider(this JobHost host)
```

Comment:
Metadata provider lifetime is just managed in the SDK with standard service injection.